lib/v2/openai/common.js

```diff
@@ -0,0 +1,72 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const getApiUrl = async () => {
```

Comment:
Double slash in https://github.com/DIYgod/RSSHub/actions/runs/4938783244/jobs/8828910028#step:12:17

This affect both `/blog/:tag?` and `/research`

Reply:
Yes, you are right. Let's fix this by using standard library function `new URL(input[, base])`.